x/mongo/driver/operation.go

```diff
@@ -871,14 +891,26 @@ func (Operation) decompressWireMessage(wm []byte) ([]byte, error) {
 }
 
 func (op Operation) createWireMessage(ctx context.Context, dst []byte,
-	desc description.SelectedServer, conn Connection) ([]byte, startedInformation, error) {
+	desc description.SelectedServer, srvr Server, conn Connection) ([]byte, startedInformation, error) {
```

Comment:
Refactored to use the function declaration described above (although `maxTimeMS` is `*uint64` to allow for unset values). Calculated `maxTimeMS` in `Execute` instead of `createWireMessage`.